packages/compile/src/model/read-variables.js

```diff
@@ -12,17 +12,21 @@ import Variable from './variable.js'
  *
  * @param {*} parsedModel TODO: Use ParsedVensimModel type here
  * @param {Object.<string, string>} [specialSeparationDims] The variable names that need to be
- * separated because of circular references.  A mapping from "C" variable name to "C" dimension
- * name to separate on.
+ * separated because of circular references.  This is an optional mapping from "C" variable name
+ * to "C" dimension name to separate on.
+ * @param {string[][]} [separateAllVarsWithDims] An optional array containing arrays of dimension
```

Comment:
I changed the type to `string[][]` to make it slightly more obvious that how this works in the case of providing more than one dimension.